advanced_functionality/distributed_tensorflow_mask_rcnn/mask-rcnn-s3.ipynb

```diff
@@ -0,0 +1,566 @@
+{
```

Comment:
<p>The link to Mask R-CNN on arxiv is broken. Instead of <a href="https://arxiv.org/abs/1703.06870_model" target="_blank">https://arxiv.org/abs/1703.06870_model</a>,  <a href="https://arxiv.org/abs/1703.06870" target="_blank">https://arxiv.org/abs/1703.06870</a> worked for me.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/awslabs/amazon-sagemaker-examples/pull/891/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>